interpreter/cling/lib/Interpreter/InvocationOptions.cpp

```diff
@@ -136,6 +136,10 @@ void CompilerOptions::Parse(int argc, const char* const argv[],
       // case options::OPT_nostdinc:
       case options::OPT_nostdincxx: NoCXXInc = true; break;
       case options::OPT_v: Verbose = true; break;
+      case options::OPT_fmodules: CxxModules = true; break;
+      case options::OPT_fmodule_name_EQ:
```

Comment:
Can you add `// explicit fall-through` here?

Reply:
We are in LLVM here, so we can even use the explicit LLVM fallthrough :). Fixed, thanks!